src/app.js

```diff
@@ -1 +1,26 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const copy = () => {
+  const args = process.argv.splice(2);
+  const [source, dest] = args;
+
+  if (!source || !dest) {
+    throw new Error('File name and file copy name are required');
+  } else if (source === dest) {
+    throw new Error('Can\'t copy in the same file');
+  }
+
+  try {
+    const data = fs.readFileSync(path.join(__dirname, source), 'utf8');
+
+    fs.writeFileSync(path.join(__dirname, dest), data, 'utf8');
```

Comment:
Not all files are utf8, don't pass this param to work with raw buffers, no extra encoding parsing. Also this will break non text files
```suggestion
    const data = fs.readFileSync(path.join(__dirname, source));

    fs.writeFileSync(path.join(__dirname, dest), data);
```